test-ets.sh

```diff
@@ -14,7 +14,7 @@ final_exit_code=0
 
 function run_and_annotate {
   echo "running retesteth $1"
-  ets/retesteth -t "$1" &> "retesteth-$1-log.txt"
```

Comment:
Could you also `tee` the output to the console so that it's possible to follow what's going on in buildkite?

Reply:
Yes I introduced that in the first commit in this branch actually, to do debugging. But with the timeout the output should eventually end up as artifact. And for developing on ETS it should probably be run locally.

What do you think?